docs/multiple-stolon.md

```diff
@@ -0,0 +1,50 @@
+# Installing multiple stolon clusters in one Gravity cluster
+This document provides an instruction on how to build and install multiple stolon clusters 
+in one Gravity cluster.
+
+## Requirements
+
+- tele binary: 5.5+
+- stolon application: 1.13.0+
+
+## Building
+By default using provided [Makefile](../Makefile) stolon application is built as a Gravity application and could be installed in Gravity cluster with `gravity install` from resulting tarball.  
+For the additional stolon(s) in cluster [Application Catalog](https://gravitational.com/gravity/docs/catalog/) feature could be reused. Application does not need to be pushed to catalog, but the process of making of that application image type applies here.
+
+Stolon application contains [Helm](https://helm.sh/) chart in `resources/charts/stolon`. Custom values are stored in [custom-values.yaml](../resources/custom-values.yaml). They are important for installation in Gravity clusters and should be used as a starting point for your customization.
```

Comment:
contains **a** [Helm]() chart ...
Custom values are important for the installation into a Gravity cluster. [custom-values.yaml]() contains a set of values that can be used as a starting point for further customizations.